Purchases/Public/RCPurchases.h

```diff
@@ -408,63 +408,91 @@ NS_SWIFT_NAME(restoreTransactions(_:));
 - (void)setPushToken:(nullable NSData *)pushToken;
 
 /**
- * TODO
- * @param adjustID
+ * Subscriber attribute associated with the Adjust Id for the user
+ * Required for the RevenueCat Adjust integration
+ *
+ * @param adjustID nil will delete the subscriber attribute
  */
 - (void)setAdjustID:(nullable NSString *)adjustID;
 
 /**
- * TODO
+ * Subscriber attribute associated with the Appsflyer Id for the user
+ * Required for the RevenueCat Appsflyer integration
+ *
+ * @param appsflyerID nil will delete the subscriber attribute
  */
 - (void)setAppsflyerID:(nullable NSString *)appsflyerID;
 
 /**
- * TODO
+ * Subscriber attribute associated with the Facebook SDK Anonymous Id for the user
+ * Recommended for the RevenueCat Facebook integration
+ *
+ * @param fbAnonymousID nil will delete the subscriber attribute
  */
 - (void)setFBAnonymousID:(nullable NSString *)fbAnonymousID;
 
 /**
- * TODO
+ * Subscriber attribute associated with the mParticle Id for the user
+ * Recommended for the RevenueCat mParticle integration
+ *
+ * @param mparticleID nil will delete the subscriber attribute
  */
 - (void)setMparticleID:(nullable NSString *)mparticleID;
 
 /**
- * TODO
+ * Subscriber attribute associated with the OneSignal Player Id for the user
+ * Required for the RevenueCat OneSignal integration
+ *
+ * @param onesignalID nil will delete the subscriber attribute
  */
 - (void)setOnesignalID:(nullable NSString *)onesignalID;
 
 /**
- * TODO
+ * Subscriber attribute associated with the install media source for the user
+ *
+ *  @param mediaSource nil will delete the subscriber attribute.
```

Comment:
super minor but there's a space before the @ here and in the following

Reply:
fixed, also fixed it in a few others before this one